### ISSUE TITLE
No predictions for articles

### ISSUE THREAD

**Issue by arguellesm** [arguellesm/faux#12]
We can't currently predict whether an article contains misleading information or not. It should be possible to do something like:

```python
article.predict()
```

to get a index representing how misleading or fake that article is.

**Comment by JJ** [comment]
Commits don't say how this is solved, namely, using an external model, apparently.

**Comment by arguellesm** [comment]
This is solved by:
- Adding a model to **predict** whether a text might be misleading or not. I included further explanation regarding the preprocessing and algorithm used in [`model.ipynb`](https://github.com/arguellesm/faux/blob/7fa7ae2b94212e4514d805a78c706c74b4463edd/model/model.ipynb), where the model is created and trained. Although the model uses external libraries, such as `nltk` or `sklearn`, the model itself doesn't come from an external source.
- Adding predictions capabilities via **blacklists**. I messed up the first commit (c9b1943) and didn't mention this issue.
